vignettes/epichains.Rmd

```diff
@@ -461,7 +462,66 @@ aggregate(sim_chains, by = "time")
 
 ### Plotting
 
-Aggregated `<epichains_tree>` objects can easily be plotted using base R
+We can plot individual chains in an `<epichains>` object using the
+[{epicontacts} package](https://www.repidemicsconsortium.org/epicontacts/).
+
+To do this, we first need to create an `<epicontacts>` object using the
+`epicontacts::make_epicontacts()` function, which requires a linelist and a
+contacts data.frame (See `?epicontacts::make_epicontacts()`).
+
+::: {.alert .alert-info}
+
+### Some notes about interoperability between `<epicontacts>` and `<epichains>` objects
+
+- For the `epicontacts::make_epicontacts()` function to work:
+  - the `linelist` and `contacts` data.frames must have a column named "id" that
+  uniquely identifies each case.
+  - The `linelist` and `contacts` objects can be the same.
+
+- An `<epichains>` object contains multiple independent chains with their own
+unique infectee ids, so this does not work out of the box with
+`make_epicontacts()` unless you use a subset chain.
+
+- For now, we will have to select and visualise individual chains from the
+`<epichains>` object.
+
+- In future versions of this package, we will make this interaction seamless
+by providing user-friendly functionality to convert an `<epichains>` object to
+an `<epicontacts>` in a way that allows us to plot all chains at once.
+
+:::
+
+Let's subset one of the chains with the biggest size from the `sim_chains` object
+created above.
+
+```{r longest-chain}
+# Get the biggest chain
+longest_chain <- sim_chains[sim_chains$chain == which.max(
+  unname(table(sim_chains$chain))
+), ]
+
+# Convert to data.frame to view the whole data
+as.data.frame(longest_chain)
+```
+
+Now, we can plot the chain using the `{epicontacts}` package.
+
+```{r plot_chain}
+# Create an `<epicontacts>` object
+epc <- make_epicontacts(
+  linelist = longest_chain,
+  contacts = longest_chain,
+  id = "infectee",
+  from = "infector",
+  to = "infectee",
+  directed = TRUE
+)
+
+# Plot the chain
+plot(epc)
+```
```

Comment:
@joshwlambert The {epicontacts} plot is not rendering here using pkgdown::build_site() even though it works in my RStudio viewer when the vignette is rendered locally. Could you help me diagnose what might be wrong?

Reply:
Thanks, Josh. It was indeed the output format. This worked for me. I'll assess the drawbacks and see if it's worth it.